src/renderers/dom/fiber/ReactDOMFiber.js

```diff
@@ -352,6 +352,10 @@ var ReactDOM = {
   },
 
   unmountComponentAtNode(container : DOMContainerElement) {
+    invariant(
+      isValidContainer(container),
+      'unmountComponentAtNode(...): Target container is not a DOM element.'
+    );t
```

Comment:
Extra `t` on the end ^

Isn't this unnecessary though?

`unmountComponentAtNode` calls `renderSubtreeIntoContainer` which calls `validateContainer` which throws- just with a slightly different error message (eg "Target container is not a DOM element.")